lib/Horde/Db/Adapter/Postgresql/Schema.php

```diff
@@ -1146,6 +1155,9 @@ public function postgresqlVersion()
                 $version = $this->selectValue('SELECT version()');
                 if (preg_match('/PostgreSQL (\d+)\.(\d+)\.(\d+)/', $version, $matches))
                     $this->_version = ($matches[1] * 10000) + ($matches[2] * 100) + $matches[3];
+                else if (preg_match('/PostgreSQL (\d\d+)\.(\d+)/', $version, $matches))
```

Comment:
This could be done in a single regexp, by making the the third dot-digit-sequence optional. And \d\d+ is not necessary.

Reply:
yes, but you would have to check later if matches[3] exists or ignore the micro version altogether.
True, \d\d+ is not really necessary, but I thought that as Postgresql changed with 10er versioning schema I could also adjust this part of RE.